packages/tools/pie-icons-webc/PieIconComponent.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    LitElement, css, PropertyValues, type TemplateResult,
+} from 'lit';
+import { property, state } from 'lit/decorators.js';
+
+import {
+    getSvgProps, RegularIconSize, LargeIconSize,
```

Comment:
I think we can do this?
```suggestion
    getSvgProps, type RegularIconSize, type LargeIconSize,
```

Reply:
🔥